### ISSUE TITLE
Normalize returned data

### ISSUE THREAD

**Issue by wilsonpage** [mozilla-magnet/magnet-metadata-service#14]
On the client I'm currently normalizing data into the following structure:

``` js
{
  url: String,
  type: String,
  title: String,
  description: String,
  image: String, // large poster image
  icon: String,
  embed: Object,
  data: Object // any type specific data
};
```

Does is make sense to do this on the service side?

**Comment by arcturus** [comment]
I think that was the original idea, also we created the endpoints with a 'raw' option to get everything from the server, or a filtered version by default.

**Comment by wilsonpage** [comment]
Fixed by #15
